src/webpack-plugin.ts

```diff
@@ -2,26 +2,43 @@ import * as assert from 'assert'
 import * as fs from 'fs'
 import * as path from 'path'
 import { Compiler } from 'webpack'
-import { generateRoutes, GenerateConfig } from 'vue-route-generator'
+import { generateRoutes } from 'vue-route-generator'
 
 const pluginName = 'VueAutoRoutingPlugin'
 
-interface Options extends GenerateConfig {}
```

Comment:
Why did you remove the `extends GenerateConfig`?

Reply:
fixed
```
interface Options extends GenerateConfig {
  pageName?: string;
}
```